### PR TITLE
test: Script updates for flaky fixes

### PR DESCRIPTION
**Pull request by Aishwarya-U-R**
## Description

> This PR includes cypress scripts fixes for stable CI runs including:

- Toasts msg validation updates
- Promises spec flaky fix
- UUID spec flaky fix
- Autocomplete JS spec flaky fix
- Query abort failure fix
- Datatree Mutation spec flaky fix
- CommunitySpec flaky fix
- AllInputWidgets spec flaky fix
- Scrolling spec flaky fix
- GetElement() helper added
- toWriteAfterToastsDisappear flag added
- Faker added

## Type of change

- Script fixes (non-breaking change which fixes failing cases)

## How Has This Been Tested?

- CI runs

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by yatinappsmith — APPROVED** [review]

File: app/client/cypress/support/Pages/AggregateHelper.ts [hunk 1]
```diff
@@ -204,19 +226,28 @@ export class AggregateHelper {
...
-      this.Sleep();
+    let locator = selector.includes("//")
+      ? cy.xpath(selector)
+      : cy.get(selector);
+    locator.waitUntil(($ele) => cy.wrap($ele).should("be.visible"), {
+      errorMsg: "Element did not appear even after 10 seconds",
+      timeout: 10000,
+      interval: 1000,
     });
+
```
Comment (mohanarpit) [inline app/client/cypress/support/Pages/AggregateHelper.ts:237]: Minor nit: Please remove commented code like this. Having a repo peppered with commented code makes it harder and more confusing for developers in the future to understand why the code was commented and if there is still a relevance to it. If we are keeping commented code for a reason, please add a comment explaining why we'd like to keep the code for a longer period of time.

Comment (Aishwarya-U-R) [reply app/client/cypress/support/Pages/AggregateHelper.ts:230]: Sure, Got i!

**Review by yatinappsmith — APPROVED** [review]

**Pull request merged**